examples/ASP.NET Core 6/ASP.NET Core 6 NLog Example/Program.cs

```diff
@@ -7,6 +7,9 @@
 {
     var builder = WebApplication.CreateBuilder(args);
 
+    builder.Logging.ClearProviders();
+    builder.Logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Trace);
```

Comment:
Think it would make sense if these lines was moved down, so it becomes:

```c#
// NLog: Setup NLog for Dependency injection
builder.Logging.ClearProviders();
builder.Logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Trace);
builder.Host.UseNLog();
```

P.S. With NLog 5.0 then `SetMinimumLevel` is no longer necessary. And if using `UseNLog(new NLogAspNetCoreOptions() { ReplaceLoggerFactory = true })` then both lines can be removed.

Reply:
I wasn't sure about upgrading the docs to NLog 5. I agree, once released in stable it should target v5. What do you think? At least the package references should specify full version names if we target v5. Unless there's a way to create wildcards for prefixes that I don't know about. Currently it looks like this:

```xml
<ItemGroup>
  <PackageReference Include="NLog.Web.AspNetCore" Version="4.*" />
  <PackageReference Include="NLog" Version="4.*" />
</ItemGroup>
```

Reply:
Yes leave the docs with `4.*` until we have NLog 5.0 RTM on the road.